MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -136,6 +160,16 @@ public void onItemClicked(RecyclerView recyclerView, int position, View view) {
       navigationView.setNavigationItemSelectedListener(this);
       navigationView.setCheckedItem(R.id.nav_basics);
     }
+
+    loggedIn = PreferenceManager.getDefaultSharedPreferences(getApplicationContext())
+      .getBoolean("TOKEN_SAVED", false);
+
+    if (loggedIn) {
+      AnalyticsTracker.getInstance(getApplicationContext()).setMapboxUsername();
+    }
+    checkForFirstTimeOpen();
+    AnalyticsTracker.getInstance(getApplicationContext()).viewedScreen("MainActivity",
```

Comment:
done. Switched to:

`analytics.viewedScreen(MainActivity.class.getSimpleName(), loggedInOrNot);`